dfhack.init-example

```diff
@@ -177,6 +181,7 @@ tweak import-priority-category
 # Misc. UI tweaks
 tweak block-labors              # Prevents labors that can't be used from being toggled
 tweak civ-view-agreement
+tweak eggs-fertile
```

Comment:
I could make it possible to toggle in the UI, I suppose (like `tweak hide-priority`).